packages/components/src/loaders/Loader.tsx

```diff
@@ -0,0 +1,7 @@
+interface LoaderProps {
+  size?: string;
+}
+
+export function Loader({ size }: LoaderProps) {
+  return <span className={`loading loading-spinner loading-${size}`}></span>;
```

Comment:
Talwind no funciona si usas variables en el nombre de las clases, porque compila solamente las frases completas antes de reemplazar las variables. Es decir loading-${size} no se compila, pero escribir textualmente "loading-sm" sí. Puedes usar un objeto para mappear los tamaños a sus respectivas clases.

```suggestion
  const loadingSizes = {
    'sm': 'loading-sm',
    'md': 'loading-md',
    'lg': 'loading-lg',
    'xl': 'loading-xl',
  }

  return <span className={cn('loading loading-spinner', loadingSizes[size])}></span>;
```